src/Lykke.Service.PayVolatility/Rabbit/TickPricesSubscriber.cs

```diff
@@ -74,10 +74,19 @@ public void Dispose()
 
         private Task ProcessMessageAsync(TickPrice tickPrice)
         {
-            if(!_assetPairs.Contains(tickPrice.Asset, StringComparer.OrdinalIgnoreCase))
-                return Task.CompletedTask;
+            try
+            {
+                if (!_assetPairs.Contains(tickPrice.Asset, StringComparer.OrdinalIgnoreCase))
+                    return Task.CompletedTask;
 
-            return _candlesRepository.AddAsync(new Candle(tickPrice));
+                return _candlesRepository.AddAsync(new Candle(tickPrice));
+            }
+            catch (Exception ex)
```

Comment:
If you don't wait any exception here than no need to catch it. If unexpected exception occurs the subscriber will handle and log it itself.

Reply:
It will log them as warnings, not errors.